spec/lib/scripts/collection_items_list_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+require "#{Rails.root}/lib/scripts/collection_items_list"
+
+RSpec.describe 'lib/scripts/collection_items_list.rb' do
+  let(:user) { FactoryGirl.create(:user) }
+  let(:file) {
+    make_generic_file(user, { id: 'bogus-file-id', title: ['Test File'] })
+  }
+  let(:parent_collection) {
+    make_collection(user, { title: 'Test Collection'})
+  }
+
+  before do
+    parent_collection.members << file
+    parent_collection.save!
+  end
+
+  describe 'CollectionItemsList' do
+    subject { CollectionItemsList.new(parent_collection.id) }
+
+    it '#initialize' do
+      expect(subject.collection).to be_truthy
```

Comment:
Is this just checking presence? If that's the case it might make more sense to use `expect(subject.collection.present?).to be_truthy` or something along those lines. This would just to make it clear that it's a test for presence only. This is very nit picky since most people coding in ruby would know the truthy values anyways so I'll leave it up to you how you want to handle this.

Reply:
I see what you mean. I think I was just testing that you can call `#collection` and `#collection_items_csv_file` on the object. It is not as clear that's what I am really doing, but checking for the presence will be.